astore/server/auth/factory.go

```diff
@@ -29,8 +28,8 @@ func DefaultFlags() *Flags {
 
 func (f *Flags) Register(set kflags.FlagSet, prefix string) *Flags {
 	set.DurationVar(&f.TimeLimit, prefix+"time-limit", f.TimeLimit, "How long to wait at most for the user to complete authentication, before freeing resources")
-	set.StringVar(&f.Principals, prefix+"principals", f.Principals, "Allowed principals and the ability to auth,comma separated string")
-	set.StringVar(&f.PathToCA, prefix+"ca", f.PathToCA, "Path to the certificate authority private file")
+	set.StringVar(&f.Principals, prefix+"principals", f.Principals, "Authorized ssh users which the ability to auth, in a comma separated string e.g. \"john,root,admin,smith\"")
```

Comment:
uhm .. now I'm confused though. When I authenticate with the auth server, I need a certificate for myself, eg, `carlo`, since I'm `carlo@enfabrica.net`. In the future, I may get `extra roles` (like root or similar) based on the group I belong to.

Are those additional principals, on top of my user name, I presume? If yes, just update the comment accordingly.
If not, let's chat :)